configure.ac

```diff
@@ -1745,6 +1762,7 @@ echo "  multiprocess  = $build_multiprocess"
 echo "  with wallet   = $enable_wallet"
 if test "x$enable_wallet" != "xno"; then
     echo "    with sqlite = $use_sqlite"
+    echo "    with bdb    = $use_bdb"
```

Comment:
Should we list bdb first since it's older?

Reply:
I don't think it matters.